tests/fixtures/addon/simple/package.json

```diff
@@ -8,7 +8,7 @@
     "paths": ["./lib/ember-super-button"]
   },
   "devDependencies": {
-    "ember-resolver": "^5.0.1",
```

Comment:
I take it all these fixtures with 5.0.1 are not even used, otherwise where does that number come from lol.

Reply:
I think it just happened to be the version at the time the fixtures were authored.

Reply:
Right but they must not be asserted against, because there was no v5.0.1 in the src/